ADAL/src/public/ADAL.h

```diff
@@ -26,6 +26,21 @@ FOUNDATION_EXPORT double ADALFrameworkVersionNumber;
 //! Project version string for ADALiOSFramework.
 FOUNDATION_EXPORT const unsigned char ADALFrameworkVersionString[];
 
+#if TARGET_OS_IPHONE
+//iOS:
```

Comment:
Fun fact: TARGET_OS_MAC is defined as true on both iOS and Mac OS, which is why !TARGET_OS_IPHONE is used to determine if we're on a Mac or not.